src/connection-manager/connections.ts

```diff
@@ -37,6 +45,21 @@ function subscribe(ws: WebSocket): void {
   )
 }
 
+/**
+ * Sends a ledger_entry WebSocket request to retrieve amendments enabled on a network.
+ *
+ * @param ws - A WebSocket object.
+ */
+function ledger_entry(ws: WebSocket): void {
```

Comment:
This shouldn't be called just `ledger_entry`, it should be something like `getAmendmentLedgerEntry` - it's not just a wrapper for _any_ `ledger_entry` request.